packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -124,15 +137,33 @@ export class VaTelephone {
     return labelPieces.join('. ');
   }
 
-  static createHref(contact: string, extension: number): string {
+  static createHref(contact: string, extension: number, sms: boolean): string {
     const isN11 = contact.length === 3;
     // extension format ";ext=" from RFC3966 https://tools.ietf.org/html/rfc3966#page-5
     // but it seems that using a comma to pause for 2 seconds might be a better
     // solution - see https://dsva.slack.com/archives/C8E985R32/p1589814301103200
-    const href = `tel:${isN11 ? contact : `+1${contact}`}`;
+    let href = `tel:+1${contact}`;
+    if (isN11) {
+      href = `tel:${contact}`;
+    } 
+    if (sms) {
+      href = `sms:${contact}`;
+    }
     return `${href}${extension ? `,${extension}` : ''}`;
   }
 
+  static createTelLabel(tty: boolean, sms: boolean): string {
+    let label = '';
+    if (tty) {
+      label = 'TTY: ';
+    }
+    if (sms) {
+      label = 'Text: ';
+    }
+    // If both tty & sms are true, we don't want to display a label.
+    return tty && sms ? '' : label;
```

Comment:
Using the if statements like this makes it less obvious that this is supposed to be mutually exclusive behavior. The return is a bit confusing as well since `label` could be an empty string, so we would end up with:

```js
    // If both tty & sms are true, we don't want to display a label.
    return tty && sms ? '' : '';
```

Can this be changed so that the output of the function is more obvious?

Reply:
This is no longer relevant since the TTY changes have been reverted.